### DISCUSSION TITLE
How to restrict users to only be able to translate specific languages?

### DISCUSSION THREAD

**Discussion by adamdewey** [WeblateOrg/weblate#8400] [Q&A]
I want to allow certain users to translate certain languages on a project but not other languages. 

How can I do this?

I tried to create a Group which only has one language and which has the Role 'Translate'. 
I then associated a user with only this Group. 

<img width="1127" alt="image" src="https://user-images.githubusercontent.com/19318937/204746025-547a428f-32d8-4917-935d-7968ba9b2259.png">

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/19318937/204745871-041a57c5-711a-4638-8c2b-04b32f47e08d.png">

I expected the user to only be able to translate the one language in the Group, however they are able to translate all languages in the project. 

How can I restrict them to just one language?

Many thanks

**Comment by nijel** [comment]
What is access control of the project? Any user will have access for public projects. Generally, look for the site-wide groups where the user is a member.

**Reply by adamdewey** [reply]
Oh thanks, I will check. By groups do you mean teams?

**Reply by nijel** [reply]
Teams are at the project level, site-wide ones are still called groups.
